docs/source/javascripts/all.js

```diff
@@ -1,2 +1,17 @@
 places({container: document.querySelector('#landing-demo')});
 
+document.addEventListener('scroll', function(e){
+	var v = e.target.scrollingElement.scrollTop;
+	var h = document.querySelector('.hero-section').offsetHeight;
```

Comment:
Can you store references to DOM Element first and then query attributes where needed?

```js
const hero = document.querySelector('.hero-section');
// use hero.*
// const navigation =
```